src/components/HomePage.js

```diff
@@ -212,6 +213,12 @@ export default React.createClass({
         return (msglink.match(/^[\!#]([^\/:]+?):(.+?)\/\$([^\/:]+?):(.+?)$/) && encodeURI(msglink) === msglink);
     },
 
+    isGroupValid(group) {
+        console.log(group);
```

Comment:
Are These console logs really needed? 😇

Reply:
Not necessarily, but isMsglinkValid did the same and I thought I'd stick with it.